specs/phase0/weak-subjectivity.md

```diff
@@ -77,51 +75,40 @@ a safety margin of at least `1/3 - SAFETY_DECAY/100`.
 
 ### Calculating the Weak Subjectivity Period
 
-A detailed analysis of the calculation of the weak subjectivity period is made in [this report](https://github.com/runtimeverification/beacon-chain-verification/blob/master/weak-subjectivity/weak-subjectivity-analysis.pdf). The expressions in the report use fractions, whereas we only use uint64 arithmetic in eth2.0-specs. The expressions have been simplified to avoid computing fractions, and more details can be found [here](https://www.overleaf.com/read/wgjzjdjpvpsd).
+A detailed analysis of the calculation of the weak subjectivity period is made in [this report](https://github.com/runtimeverification/beacon-chain-verification/blob/master/weak-subjectivity/weak-subjectivity-analysis.pdf).
 
-#### `get_active_validator_count`
+*Note*: The expressions in the report use fractions, whereas eth2.0-specs uses only `uint64` arithmetic. The expressions have been simplified to avoid computing fractions, and more details can be found [here](https://www.overleaf.com/read/wgjzjdjpvpsd).
 
-```python
-def get_active_validator_count(state: BeaconState) -> uint64:
-    active_validator_count = len(get_active_validator_indices(state, get_current_epoch(state)))
-    return active_validator_count
-```
-
-#### `compute_avg_active_validator_balance`
-
-```python
-def compute_avg_active_validator_balance(state: BeaconState) -> Ether:
-    total_active_balance = get_total_active_balance(state)
-    active_validator_count = get_active_validator_count(state)
-    avg_active_validator_balance_gwei = total_active_balance // active_validator_count
-    avg_active_validator_balance_eth = avg_active_validator_balance_gwei // ETH_TO_GWEI
-    return avg_active_validator_balance_eth
-```
+*Note*: The calculations here use `Ether` instead of `Gwei`, because the large magitude of balances in `Gwei` can cause an overflow while computing using `uint64` arithmetic operations. Using `Ether` reduces the magintude of the multiplicative factors by an order of `ETH_TO_GWEI` (`= 10**9`) and avoid the scope for overflows in `uint64`.
```

Comment:
```suggestion
*Note*: The calculations here use `Ether` instead of `Gwei`, because the large magnitude of balances in `Gwei` can cause an overflow while computing using `uint64` arithmetic operations. Using `Ether` reduces the magnitude of the multiplicative factors by an order of `ETH_TO_GWEI` (`= 10**9`) and avoid the scope for overflows in `uint64`.
```